datasets/cn/peoples_congress_wikipedia/crawler.py

```diff
@@ -124,17 +172,25 @@ def crawl(context: Context):
     ids = defaultdict(int)
 
     for h3 in doc.findall(".//h3"):
-        delegation_match = REGEX_DELEGATION_HEADING.match(h3.text_content())
-        if not delegation_match:
+        h3_text = h3.text_content().strip()
+        delegation_match = REGEX_DELEGATION_HEADING.match(h3_text)
+        # Determine whether to process the <h3> based on delegation match or specific headings
+        if delegation_match:
+            delegation_name = delegation_match.group(1)
+            table = h3.getparent().getnext().getnext()
+        elif any(heading in h3_text for heading in CHANGES_IN_REPRESENTATION):
+            delegation_name = None
+            table = h3.getparent().getnext()
+        else:
             continue
-        delegation_name = delegation_match.group(1)
-        table = h3.getparent().getnext().getnext()
+
         if table.tag != "table":
             table = table.getnext()
         assert table.tag == "table"
         for row in parse_table(context, table):
             id = crawl_item(context, row, delegation_name)
-            ids[id] += 1
+            ids[id] = ids.get(id, 0) + 1
```

Comment:
ids is a `defaultdict` initialising values using `int()` i.e. `0` so we don't have to initialise it here

```suggestion
            ids[id] += 1
```